packages/avatars/src/styled/StyledStandaloneStatusIndicator.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { math } from 'polished';
+
+import { IStatusIndicatorProps, STATUS } from '../types';
+import { getStatusColor, statusIconStyles, TRANSITION_DURATION } from './utility';
+
+const COMPONENT_ID = 'avatars.status-indicator.indicator';
+
+const [available, away, transfers, offline] = STATUS;
+
+const sizeStyles = (props: IStatusIndicatorProps & ThemeProps<DefaultTheme>) => {
+  const borderWidth = props.theme.shadowWidths.sm;
+
+  let size = '0';
+  let marginTop = props.theme.space.base;
+
+  if (props.isCompact) {
+    marginTop = props.theme.space.base * 1.5;
+    size = math(`${props.theme.space.base * 3}px - (${borderWidth} * 2)`);
+  } else {
+    size = math(`${props.theme.space.base * 4}px - (${borderWidth} * 2)`);
+  }
+
+  return css`
+    margin: ${props.theme.space.base}px;
```

Comment:
What is the impetus for margining the image? We don't see this in the Figma designs. Either the code should match the designs, or the designs should be pushed to explore inline `StatusIndicator` layouts similar to https://garden.zendesk.com/components/span#icons.

Reply:
None in particular, I will update.